--- gnustep-make-1.5.1/configure.ac	Fri Oct  4 15:56:00 2002
+++ gnustep-make-1.5.1.new/configure.ac	Sat Dec 21 20:13:44 2002
@@ -465,7 +465,7 @@
 #--------------------------------------------------------------------
 # Produce the output files
 #--------------------------------------------------------------------
-AC_CONFIG_FILES([config.h config.make openapp debugapp opentool 
+AC_CONFIG_FILES([config.make openapp debugapp opentool 
 executable.template GNUmakefile GNUstep.sh GNUstep.csh fixpath.sh
 gnustep-make.spec Documentation/GNUmakefile])
 AC_CONFIG_COMMANDS([default],
