diff -Nur gnustep-make-2.0.2.orig/target.make gnustep-make-2.0.2.no-chain-library-links/target.make
--- gnustep-make-2.0.2.orig/target.make	2007-11-07 11:56:37.000000000 -0700
+++ gnustep-make-2.0.2.no-chain-library-links/target.make	2008-02-02 21:02:58.000000000 -0700
@@ -546,7 +546,7 @@
             $(RM_LN_S) $(LIB_LINK_SONAME_FILE);\
             $(LN_S) $(LIB_LINK_VERSION_FILE) $(LIB_LINK_SONAME_FILE); \
           fi; \
-          $(LN_S) $(LIB_LINK_SONAME_FILE) $(LIB_LINK_FILE); \
+          $(LN_S) $(LIB_LINK_VERSION_FILE) $(LIB_LINK_FILE); \
 	)
 AFTER_INSTALL_SHARED_LIB_CMD = \
 	(cd $(LIB_LINK_INSTALL_DIR); \
