--- gnustep-make-1.0.1/GNUstep.csh.in.orig	Sat May  5 12:44:11 2001
+++ gnustep-make-1.0.1/GNUstep.csh.in	Fri Oct 12 15:42:56 2001
@@ -72,9 +72,10 @@
 
 set tp = ""
 foreach dir ( `/bin/sh -c 'IFS=:; for i in ${GNUSTEP_PATHPREFIX}; do echo $i; done'` )
-  set tp=${tp}${dir}/Tools:
+  echo $PATH | grep '\(:\|^\)'${dir}/Tools'\(:\|$\)' || set tp=${tp}${dir}/Tools:
   if ( ! ${?GNUSTEP_FLATTENED} ) then
-    set tp=${tp}${dir}/Tools/${GNUSTEP_HOST_CPU}/${GNUSTEP_HOST_OS}:
+    echo $PATH | grep '\(:\|^\)'${dir}/Tools/${GNUSTEP_HOST_CPU}/${GNUSTEP_HOST_OS}'\(:\|$\)' \
+	|| set tp=${tp}${dir}/Tools/${GNUSTEP_HOST_CPU}/${GNUSTEP_HOST_OS}:
   endif
 end
 setenv GNUSTEP_PATHPREFIX_LIST $GNUSTEP_PATHPREFIX
